### PR TITLE
[Jupyter] [Shell] New jupyter 3.4.8

### PR DESCRIPTION
**Pull request by danykaufman**
### Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove irrelevant fields.]

- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

### Description

[Please add a description of the change request.]

### CODE REVIEW

File: stable/jupyter/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-version: 0.14.12
+version: 0.14.13
```
Comment (liranbg) [inline stable/jupyter/Chart.yaml:1]: bump minor, will you

File: stable/jupyter/templates/jupyter-configmap.yaml [hunk 2]
```diff
@@ -53,18 +56,18 @@ data:
...
-    if [ "\${CLIENT_MLRUN_VERSION}" = "\${SERVER_MLRUN_VERSION}" ] || [ "\${CLIENT_MLRUN_VERSION}" = "\${SERVER_MLRUN_VERSION//-}" ]; then
-      echo "Both server & client are aligned (\${CLIENT_MLRUN_VERSION})."
-    else
-      if [ \${CLIENT_MLRUN_VERSION} ]; then
-        pip uninstall -y mlrun
-      fi
-      pip install mlrun[complete]==\${SERVER_MLRUN_VERSION}
-    fi
-    EOF
+      echo "CLIENT_MLRUN_VERSION=\`pip show mlrun | grep Version | awk '{print \$2}'\`" >> ${ALIGN_MLRUN_SCRIPT}
```
Comment (liranbg) [inline stable/jupyter/templates/jupyter-configmap.yaml:59]: why not using @AlonMaor14 way of `cat` and inline bash commands without need to add eco everywhere? that way indentation is much clearer

File: stable/jupyter/Chart.yaml (hunk 1 above)
Comment (danykaufman) [reply stable/jupyter/Chart.yaml:1]: so 0.15.0 ?

File: stable/jupyter/templates/jupyter-configmap.yaml (hunk 2 above)
Comment (danykaufman) [reply stable/jupyter/templates/jupyter-configmap.yaml:59]: Just wanted to leave it the same as other parts of the configmap where echo is being used.
In some places there are for loops generating lines so cat is not an option

Comment (danykaufman) [reply stable/jupyter/templates/jupyter-configmap.yaml:59]: I can try and replace most of the config map to the cat way but let's do it in a later commit

**Pull request merged**